test/install.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# This script installs Kurator cluster-operater and fleet-manager.
+
+KUBECONFIG_PATH=${KUBECONFIG_PATH:-"${HOME}/.kube"}
+MAIN_KUBECONFIG=${MAIN_KUBECONFIG:-"${KUBECONFIG_PATH}/kurator-host.config"}
+export KUBECONFIG=${MAIN_KUBECONFIG}
+VERSION=${VERSION:-"0.6.0"}
```

Comment:
We should build from code

Reply:
I get it. I need to pull the latest code commits, and use the built images and chart packages to install.